tests/acceptance/init.spec.js

```diff
@@ -53,7 +53,7 @@ describe('Acceptance: ng init', function() {
         });
 
         expected.forEach(function (file, index) {
-            expected[index] = file.replace(/__name__/g, 'angular-cli');
+            expected[index] = file.replace(/__name__/g, 'tmp');
```

Comment:
If the API changed shouldn't we switch the implementation rather than changing the test criteria?